### DISCUSSION TITLE
Adding one custom link in the console?

### DISCUSSION THREAD

**Discussion by maxximino** [zitadel/zitadel#6984] [Ideas]
Hi,
when users end up in the console (e.g. to change password), they are "stuck there" and don't have any link for going back to the "main application" (i.e. what zitadel is managing access for).
I tried to customize the footer links - but those don't apply to the console (only to the login page).
It would be enough for me to have just one link somewhere for the whole instance/same for all users, but I can't find a way.
No strong opinions on where/how it looks.
I do realize you recommend building a customized UI for editing the user profile, but for my use-case (share a few webapps with a handful of people instead of exposing them to the whole internet) it's more than good enough, except for this little detail.
Even https://zitadel.com/docs/self-hosting/manage/configure doesn't suggest anything to me.

Am I missing some existing feature? Shall I file a feature request as an issue?

**Comment by hifabienne** [comment]
I think this related to the following issue: https://github.com/zitadel/zitadel/issues/4586
Currently the issue is in the product backlog and an external contributor mentioned to take over but I do not know the state.
From our side the priority is not as high, this might change in the future. So if you want to see it implemented please upvote.

**Comment by maxximino** [comment]
Thanks! Upvoted. If someone sees this thread and knows a workaround to sneak in even just a single link (...without having to rebuild the container!) please post it!
